R/learn.R

```diff
@@ -41,6 +41,130 @@ dm_learn_from_db <- function(dest, dbname = NULL, ...) {
     return()
   }
 
+  if (!is_mssql(con)) {
```

Comment:
AFAICT, `dm_meta()` works well for PG too. We could learn from PG using the function as well in this PR.